CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+
+## [0.1.0a3] - 2021-04-23
```

Comment:
Has any of this actually been released in PyPi yet? Doesn't seem so. I would categorize all of this under one entry until it has actually been released. Can include the `Unreleased` classification for now.